backend/models/dynamodb/Entity.js

```diff
@@ -0,0 +1,87 @@
+
+import DynamoDB from "@klaudsol/commons/lib/DynamoDB"
+import { 
+  DYNAMO_DB_TYPES, 
+  DYNAMO_DB_INDEXES, 
+  DYNAMO_DB_TABLE, 
+  DYNAMO_DB_ORDER 
+} from "@/constants";
+
+export default class Entity {
```

Comment:
So I'm getting dizzy with the jargons and terms. Let's define them!

Let's forgo `Entity`. This is a relic of the `Entity-Attributes-Value` pattern that we used for RDS, but not applicable anymore in the NoSQL world. So here are the some terms:

1.) 

```suggestion
export default class Content {
```